src/commissaire_service/investigator/__init__.py

```diff
@@ -187,18 +191,20 @@ def on_investigate(self, message, address, cluster_data={}):
             # Save the updated host model.
             params = {
                 'model_type_name': host.__class__.__name__,
-                'model_json_data': host.to_json()
+                'model_json_data': host.to_json(secure=True)
             }
             self.request('storage.save', params=params)
 
         # Verify association with relevant container managers
-        params = {
-            'cluster_type': cluster.type,
-            'address': address
-        }
-        response = self.request('storage.node_registered', params=params)
-        if response['result']:
-            host.status = 'active'
+        # FIXME Adapt this for ContainerManagerConfig.
+        # params = {
+        #     'cluster_type': cluster.type,
+        #     'address': address
+        # }
+        # response = self.request('storage.node_registered', params=params)
+        # if response['result']:
+        #     host.status = 'active'
+        host.status = 'active'  # XXX Temporary hack.
```

Comment:
I'd say set the status to ``disassociated`` to keep it inline with the current ``enums`` for statuses. Once we add in the registration we'll update this.